app/main/db/queries.py

```diff
@@ -204,8 +205,10 @@ def _build_series_filter_query(series_id):
     return query
 
 
-def _build_consignment_filter_query(
-    consignment_id: uuid.UUID, date_range=None, date_filter_field=None
+def _build_consignment_view_query(
```

Comment:
I think it could be worth making a few changes to help us with testing maintainability and readability:

- make `_build_consignment_filters` a public function `build_consignment_filters` which we can then test all the edge case combination of filters in.

- make `_build_consignment_sorting_orders` a public function `build_consignment_sorting_orders` which we can then test all the edge case combination of sorting in.

- make `_build_consignment_view_query` a public function, `build_consignment_view_query` which we write 1 or 2 integration tests.

- then only need 1 or 2 top level browse_consignment tests which care about user permissions etc. but not too bothered by all the edge cases which will be dealt progressively at lower levels.

This will help us keep more in line with the test pyramid, where there are more unit tests and fewer, more complex integration tests; in particular this will help since most of the new tests we are adding to the browse_consignment level have a huge load of test setup to cover all the edge cases and are harder to debug.

Reply:
spoken on slack, will create a refactor  ticket (or 2) for this.